src/gui/uithememanager.h

```diff
@@ -34,6 +34,7 @@
 #include <QIcon>
 #include <QObject>
 #include <QString>
+#include <QtGlobal>
```

Comment:
Let place "Qt*" headers before "Q*" ones. These are different prefixes, so I still think it is incorrect to sort them just alphabetically.

Reply:
Done.